src/Components/NewAddressDialog/NewAddressDialog.css

```diff
@@ -0,0 +1,34 @@
+.address-form{
+    min-height: 0;
+
+    display: block;
+    background-color: hsl(var(--clr-dark));
+    color: hsl(var(--clr-white));
+    padding: 2rem;
+    opacity: 90%;
+
+}
+
+.new-address-dialog{
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    backdrop-filter: blur(2px) brightness(30%);
+    z-index: 9999;
+    inset: 0% 0% 0% 0%;
+}
+
+.submit-btn{
+    padding: 0.35rem 1rem ;
+}
+
+@media (min-width: 40rem){
```

Comment:
fix: use @media only screen and max-width syntax

Reply:
it is mobile first